### ISSUE TITLE
AI

### ISSUE THREAD

**Issue by TejasLamba2006** [TejasLamba2006/Raven-bS-Docs#5]

**Comment by TejasLamba2006** [comment]
@coderabbitai
scan https://github.com/TejasLamba2006/Raven-bS files and folders and create issues for documentation that havnt been documented in this repo

**Comment by SleepingPig35** [comment]
@TejasLamba2006 you need to update the commits in your fork.

**Comment by TejasLamba2006** [comment]
@coderabbitai create issues for only the dir src/main/java/keystrokesmod/module/impl

also rescan the repo, and scan this repo as well create issues for modules that arnt documented and copy the docs style and in the issue add a your version of the documentation after analysing the .java file of the module

you can create issues directly

**Comment by TejasLamba2006** [comment]
@coderabbitai Yes start creating issues, also in the description copy the documentation style i used a propose a docs in it

**Comment by TejasLamba2006** [comment]
@coderabbitai yes this format is good but keep this as format, also after this start creating issues dont ask more confirmations

path of the hack: anything
```
mdx docs for that hack with sequence diagram thats not complicated
```
how much time it took you for making it

**Comment by TejasLamba2006** [comment]
@coderabbitai start creating issues pls

**Comment by TejasLamba2006** [comment]
@coderabbitai I talked with your developer give me the logs of the process i told u to do so i can identify the error

**Comment by TejasLamba2006** [comment]
@coderabbitai okay i understood the error firstly just create issues with the module name as tittle and keep description empty then in the same issue try sending the message with proposed documentation format, 

now create one issue right now with this instruction and let me verify if issue has been created

**Comment by TejasLamba2006** [comment]
@cderoabbitai yes the issue was created now create all issues in this format but remember to put mdx content in codeblock

**Comment by TejasLamba2006** [comment]
@coderabbitai yes the issue was created now create all issues in this format but remember to put mdx content in codeblock

**Comment by TejasLamba2006** [comment]
@coderabbitai now for the first 10 modules send another message with the documentation that would be optimized to add to the repo, u can scan the code again take your time. the documentation style should be same throughout the repo and make sequence diagrams as well
https://github.com/TejasLamba2006/Raven-bS-Docs/issues/35
https://github.com/TejasLamba2006/Raven-bS-Docs/issues/36
https://github.com/TejasLamba2006/Raven-bS-Docs/issues/37
https://github.com/TejasLamba2006/Raven-bS-Docs/issues/38
https://github.com/TejasLamba2006/Raven-bS-Docs/issues/39
https://github.com/TejasLamba2006/Raven-bS-Docs/issues/40
https://github.com/TejasLamba2006/Raven-bS-Docs/issues/41
https://github.com/TejasLamba2006/Raven-bS-Docs/issues/42
https://github.com/TejasLamba2006/Raven-bS-Docs/issues/43
https://github.com/TejasLamba2006/Raven-bS-Docs/issues/44

**Comment by TejasLamba2006** [comment]
@coderabbitai change the issue tittle to from `Document Sprint Module` to `Movement Sprint Module` i.e `Category moduleName`, also add tags accordingly for categories to issues and tags like in-prograss ets

**Comment by TejasLamba2006** [comment]
@coderabbitai change the issue tittle to from `Document Sprint Module` to `Movement Sprint Module` i.e `Category moduleName`, also add tags accordingly for categories to issues and tags like in-prograss ets. do this for each issue

**Comment by TejasLamba2006** [comment]
@coderabbitai change the issue tittle to from `Document Sprint Module` to `Movement Sprint Module` i.e `Category moduleName`, also add tags accordingly for categories to issues and tags like in-prograss ets. do this for each issue on their respective issue you created. do not do this for this issue

**Comment by TejasLamba2006** [comment]
@coderabbitai dont change name of this issue, now change name of issues that have document in it and replcae with category and add tags

**Comment by TejasLamba2006** [comment]
@coderabbitai these dont have tittle in `Category moduleName` and dont have tags as well
#70
#71 
#72 
#7 
#6 
#60 #61 

these are just examples make sure every issue except this has a tag and tittle in that format

**Comment by TejasLamba2006** [comment]
@coderbbitai make me a list of all issue sorted in category, if u find a issue that has Document in its tittle rename it and add tags

**Comment by TejasLamba2006** [comment]
@coderabbitai make me a list of all issue sorted in category, if u find a issue that has Document in its tittle rename it and add tags

**Comment by TejasLamba2006** [comment]
@coderabbitai these dont have tittle in Category moduleName and dont have tags as well 
#43 #42 #40 #39 #38 #37 #34 #3 #32 #31 #30 #29 #28 #27 #26 #25 #24 #23 #22 #21 #20 #19 #18 #17 #16

**Comment by TejasLamba2006** [comment]
@coderabbitai these dont have tittle in Category moduleName and dont have tags as well 
#41 #33 #27 #26 #25 #24 #23 #22 #21 #20 #19 #18 #17 #16

**Comment by TejasLamba2006** [comment]
@coderabbitai these dont have tittle in Category moduleName and dont have tags as well 
#19 #18 #17 #16 #15 #14 #13 #12 #11 #10 #9 #8 #7

**Comment by TejasLamba2006** [comment]
@coderabbitai not get me a list of all undocumented combat modules the format should be in a table like this
#56 | src/main/java/keystrokesmod/module/impl/combat/Velocity.java

**Comment by TejasLamba2006** [comment]
@coderabbitai Now step by step go to these paths from the repo https://github.com/TejasLamba2006/Raven-bS
#49 | src/main/java/keystrokesmod/module/impl/combat/JumpReset.java
#41 | src/main/java/keystrokesmod/module/impl/combat/ClickAssist.java
#45 | src/main/java/keystrokesmod/module/impl/combat/Criticals.java
#40 | src/main/java/keystrokesmod/module/impl/combat/AimAssist.java
#53 | src/main/java/keystrokesmod/module/impl/combat/Reduce.java
#51 | src/main/java/keystrokesmod/module/impl/combat/MoreKB.java
#46 | src/main/java/keystrokesmod/module/impl/combat/HitBox.java
#55 | src/main/java/keystrokesmod/module/impl/combat/TimerRange.java
#43 | src/main/java/keystrokesmod/module/impl/combat/BurstClicker.java
#47 | src/main/java/keystrokesmod/module/impl/combat/HitSelect.java
#54 | src/main/java/keystrokesmod/module/impl/combat/RodAimbot.java
#52 | src/main/java/keystrokesmod/module/impl/combat/Reach.java
#56 | src/main/java/keystrokesmod/module/impl/combat/Velocity.java
#50 | src/main/java/keystrokesmod/module/impl/combat/KillAura.java
#14 | src/main/java/keystrokesmod/module/impl/combat/AutoClicker.java
#42 | src/main/java/keystrokesmod/module/impl/combat/BlockHit.java
#48 | src/main/java/keystrokesmod/module/impl/combat/InfiniteAura.java


and after analaysing the code and ready this repos docs style. send a message in each issue containing a code for `.mdx` file that is production ready

**Comment by TejasLamba2006** [comment]
@coderabbitai memorize that soon am gonna push these changes to the repo u can close issues when the docs for those hacks exists

**Comment by TejasLamba2006** [comment]
@coderabbitai I am participating in an hackathon and this is the data i got from the goverment website, keep in mind this isnt related to this repostary so dont use anything from here

PS NUMBER: SIH1700

ORGANISATION: Ministry of Law & Justice

THEME: Smart Automation

Title: 

Developing an AI based interactive Chatbot or virtual assistant for the Department of Justice’s Website.

Description: Background: Department of Justice (DoJ) headed by the Secretary is a part of Ministry of Law & Justice, Government of India. Along with the functions of the Department of Justice as per the Allocation of Business (Rules), 1961, implements important schemes for Development of Infrastructure Facilities for Judiciary, setting up of Special Courts for speedy trial and disposal of cases of sensitive nature (Fast Track Special Court for cases of rape and POCSO Act), eCourts Project on computerization of various courts across the country, legal aid to poor and access to justice, financial assistance to National Judicial Academy for providing training to the Judicial Officers of the country. • Description: The above problem statement envisages a Chatbot or virtual assistant be developed to understand the user and allow them to ask questions and get information related to DoJ such as o Know about the various divisions of DoJ, o Number of Judges appointed at Supreme Court, High Courts, District & Subordinate Courts and current vacancies. o Pendency of cases through National Judicial Data Grid (NJDG) o Procedure to pay fine of traffic violation o Live Streaming of Court Cases o Steps for the eFiling and ePay o Know about working Fast track courts o Ways to download eCourts Services Mobile app o Availing Tele Law Services o Know current status of case This chatbot should be able to learn over time to add excellent value to customer interactions and should be capable for handling large data sets if scope expanded. • Expected Solution: An interactive Chatbot or virtual assistant be developed for the Department of Justice website resulting into desired information as per the command.

Dataset Link: 

https://doj.gov.in/

# CHATBOT NAME

- **VidhiVaani** - "Vidhi" means law in Sanskrit, and "Vaani" means voice, implying "Voice of Law."

Now I need you to create a flowchart how the ai chatbot would work, and a theory on how it gonna function and also how  we should make it also give links how u got this info from and more
